com.ibm.streamsx.topology/com.ibm.streamsx.topology.python/namespace-info.spl

```diff
@@ -323,29 +323,52 @@ To avoid name conflicts, do not create modules with the same name as
   The callable must return an iterable or None, otherwise a TypeError is raised.
   The callable is invoked for each incoming tuple.
 
-  return: A stream containing transformed tuples.
+  return: A Stream containing transformed tuples.
   
-* `isolate` function on `Stream`
+* `isolate(self)` function on `Stream`
 
   Guarantees that the upstream operation will run in a separate process from the downstream operation when 
   the application is executed in distributed mode.
   
-* `low_latency` and `end_low_latency` functions on `Stream`
+  return: Stream
+  
+* `low_latency(self)` function on `Stream`
 
   The function is guaranteed to run in the same process as the
   upstream Stream function. All streams that are created from the returned stream 
   are also guaranteed to run in the same process until end_low_latency() 
   is called.
+  
+  return: Stream
+  
+* `end_low_latency(self)` function on `Stream`
 
-* `union` function on `Stream`
+  Returns a Stream that is no longer guaranteed to run in the same process
+  as the calling stream.
+  
+  return: Stream
 
-  The Union operator merges the outputs of the streams in the set into a single stream.
+* `union(self, streamSet)` function on `Stream`
 
-*  `parallel` and `end_parallel` functions on `Stream`
+  Merges the outputs of the streams in the set into a single stream.
 
-  Marks operator as parallel output with width specified
+  param `streamSet`: a set of Stream objects to merge
+  
+  return: Stream
+        
+* `parallel(self, width)` function on `Stream`
 
+  Marks operator as parallel output with specified width
```

Comment:
Need to explain what parallel output is. Descriptions can be copied from the equivalent Java methods.